packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -368,7 +374,12 @@ export const getStyles = memoizeFunction(
       },
 
       optionsContainer: {
-        display: 'block'
+        display: 'block',
+        selectors: {
+          '.ms-Checkbox:hover, .ms-Checkbox:hover:active': {
+            backgroundColor: 'transparent !important'

```

Comment:
Why do we need the `!important` specificity override in here?

Reply:
I'll pull that out and get specificity elsewhere, thanks!